scripts/run_dpo.py

```diff
@@ -227,6 +218,16 @@ def main():
         trainer.model.config.use_cache = True
         trainer.model.config.save_pretrained(training_args.output_dir)
 
+    ##########
+    # Evaluate
```

Comment:
I found it better to put the evaluation after saving the model because sometimes you get OOM and cry a little bit